packages/examples/sample_app/README.md

```diff
@@ -0,0 +1,16 @@
+# sample_app
```

Comment:
Should we update this with instructions for running the sample app?

Reply:
Yeah good call, updated it.